setup2/scripts/phase2_chunked.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash -e
+
+rm -f challenge* response* new_challenge* new_response* new_new_challenge_* processed* initial_ceremony* response_list* combined* seed* chunk*
+
+export RUSTFLAGS="-C target-feature=+bmi2,+adx"
+CARGO_VER=""
+PROVING_SYSTEM=groth16
+POWER=18
+BATCH=131072
+CHUNK_SIZE=131072
+CURVE="bw6"
+PATH_PHASE1="../../phase1-cli/scripts/combined" 
+SEED1=$(tr -dc 'A-F0-9' < /dev/urandom | head -c32)
+echo $SEED1 > seed1
+SEED2=$(tr -dc 'A-F0-9' < /dev/urandom | head -c32)
+echo $SEED2 > seed2
+
+function check_hash() {
+  test "`xxd -p -c 64 $1.hash`" = "`b2sum $1 | awk '{print $1}'`"
+}
+
+cargo $CARGO_VER build --release --bin setup2
+
+phase2_new="cargo run --release --features cli  -- new --curve-type $CURVE --chunk-size $CHUNK_SIZE --batch-size $BATCH --contribution-mode full"
+phase2_chunked="cargo run --release --bin setup2 --features cli  -- --curve-type $CURVE --chunk-size $CHUNK_SIZE --batch-size $BATCH --contribution-mode full --proving-system $PROVING_SYSTEM"
```

Comment:
This should come with an argument for the binary, right? That is, one of `new`, `contribute`, etc?

Reply:
Looks like we're still missing the `seed` and the `proving_system` in the related struct.

Reply:
The command is actually specified lower in the script.